package.json

```diff
@@ -28,6 +28,7 @@
     ],
     "**/*": "prettier --write --ignore-unknown"
   },
+  "packageManager": "pnpm@8.15.8+sha512.d1a029e1a447ad90bc96cd58b0fad486d2993d531856396f7babf2d83eb1823bb83c5a3d0fc18f675b2d10321d49eb161fece36fe8134aa5823ecd215feed392",
```

Comment:
Is the sha code essential here?  Is that to be extra sure? Why not just lock to the semver version `8.15.8`?

Reply:
Okay I see now that's just what running `corepack use pnpm@latest` does, it populates `packageManagers` in package.json.  And will do next time when we want to update the version used by the project.